data/src/main/java/com/linkedin/data/schema/PdlBuilder.java

```diff
@@ -51,6 +53,7 @@
   ));
   private static final char ESCAPE_CHAR = '`';
   private static final Pattern IDENTIFIER_CHARS = Pattern.compile("[0-9a-zA-Z_-]*");
+  private static final int DEFAULT_BUFFER_SIZE = 4096;
```

Comment:
This might cause confusion, since the compact PDL builder has its own concept of a "buffer". I suggest renaming to `DEFAULT_JSON_CODEC_BUFFER` or something.

Reply:
Done!